source/material_model/simple.cc

```diff
@@ -178,6 +178,7 @@ namespace aspect
         return true;
       else
         return false;
+   
```

Comment:
Can you remove this one line again? It gives the auto-indentation script fits because you have spaces there and it doesn't want them there. The line is also unnecessary :-)